internal/component/beyla/ebpf/beyla_linux.go

```diff
@@ -285,13 +334,36 @@ func (a *Arguments) Convert() (*beyla.Config, error) {
 	if err != nil {
 		return nil, err
 	}
+	cfg.Prometheus = a.Prometheus.Convert()
+	cfg.NetworkFlows = a.Network.Convert()
+	if a.Debug {
+		cfg.SetDebugMode()
+	}
 	return &cfg, nil
 }
 
 func (args *Arguments) Validate() error {
 	if args.Port == "" && args.ExecutableName == "" && len(args.Discovery.Services) == 0 {
 		return fmt.Errorf("you need to define at least open_port, executable_name, or services in the discovery section")
 	}
+	validInstrumentations := map[string]struct{}{"*": {}, "http": {}, "grpc": {}, "redis": {}, "kafka": {}, "sql": {}}
+	for _, instrumentation := range args.Prometheus.Instrumentations {
+		if _, ok := validInstrumentations[instrumentation]; !ok {
+			return fmt.Errorf("invalid prometheus.instrumentations entry: %s", instrumentation)
```

Comment:
what happens if I specify multiple times the same instrumentation? For example ['"http", "http"]

Reply:
(same question for the features below)

Reply:
They will be just ignored. Internally is treated as a set.